Tribes/src/app/building-type/building-type.component.html

```diff
@@ -17,8 +17,8 @@
           </div>
           <div *ngIf="towhalllevel !== 3" class="text-center mt-3">
             <button class="btn btn-warning"
-                [disabled]="getUpgardeingCost(townhallLevel) > goldAmount">Upgrade<br><small>for
-                  {{getUpgardeingCost(townhallLevel)}} gold</small></button>
+                [disabled]="getUpgradingCost(townhallLevel) > goldAmount">Upgrade<br><small>for
```

Comment:
where we use this component?

Reply:
Viola asked me to correct typo on this component. Please ask her about it.